### PR TITLE
Add a sample for DocumentAI

### PR DESCRIPTION
**Pull request by asrivas**

### CODE REVIEW

File: apps-script/documentai/documentai.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * Authorization to DocAI requires a GCP service account with a Document AI role.
+ */
+// Sets variable for service account privat3e key.
+var PRIVATE_KEY = '';
+ 
+// Sets variable for service account email address.
+var CLIENT_EMAIL = 'YOUR-SERVICE-ACCOUNT@appspot.gserviceaccount.com';
+ 
+function processDocument(docBytes) {
+    var service = getService();
```
Comment (sqrrrl) [inline apps-script/documentai/documentai.js]: Inconsistent formatting (indentation)

File: apps-script/documentai/documentai.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  if (service.hasAccess()) {
```
Comment (sqrrrl) [inline apps-script/documentai/documentai.js]: Suggest inverting this and making it a guard clause. E.g.

if (!service.hasAccess()) {
  Logger.log(service.getLastError();
  return;
}
// rest of code here...

File: apps-script/documentai/documentai.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+const PROCESSOR_ID = ''; // Create processor in Cloud Console
+ 
+const TEST_FILE_ID = ''; // Drive ID of an image or pdf to use for testing
+ 
+/**
+ * Creates variables for the service account key and email address.
+ * Authorization to DocAI requires a GCP service account with a Document AI role.
+ */
+// Sets variable for service account privat3e key.
+var PRIVATE_KEY = '';
```
Comment (sqrrrl) [inline apps-script/documentai/documentai.js]: Use let instead of var (here and elsewhere)

Comment (sqrrrl) [reply apps-script/documentai/documentai.js]: Here it could probably be const since it looks like folks are expected to pre-fill this

**Review by sqrrrl — CHANGES_REQUESTED** [review]
Mostly good, suggest running through a formatter to fix some indentation/spacing inconsistencies.

File: apps-script/documentai/documentai.js (hunk 3 above)
Comment (asrivas) [reply apps-script/documentai/documentai.js]: done below as well

File: apps-script/documentai/documentai.js (hunk 2 above)
Comment (asrivas) [reply apps-script/documentai/documentai.js]: good catch!!

File: apps-script/documentai/documentai.js (hunk 1 above)
Comment (asrivas) [reply apps-script/documentai/documentai.js]: I should really add eslint in this subdirectory

**Pull request merged**